ui/components/Icon.tsx

```diff
@@ -20,6 +20,8 @@ import PauseIcon from "@material-ui/icons/Pause";
 import PersonIcon from "@material-ui/icons/Person";
 import PlayIcon from "@material-ui/icons/PlayArrow";
 import RemoveCircleIcon from "@material-ui/icons/RemoveCircle";
+import { CallReceived, CallMade, Remove } from "@material-ui/icons";
```

Comment:
You'll get  a warning for **imported multiple times**
```
import CallMade from "@material-ui/icons/CallMade";
import CallReceived from "@material-ui/icons/CallReceived";
import Remove from "@material-ui/icons/Remove";
```

Reply:
no warnings, but I changed them to be consistent with others.